README.md

```diff
@@ -35,13 +35,37 @@ chpl/chpl-service/src/test/resources/environment.test.properties
 
 ## Properties files
 
-Copy over the two template files and fill in the keylocation, downloadedFolderPath, and datasourcepassword
+The following Properties Files exist within the CHPL Resources project.
 
 ```
-chpl/chpl-auth/src/main/resources/environment.auth.properties
-
-chpl/chpl-service/src/main/resources/environment.properties
+chpl/chpl-resources/src/main/resources/email.properties
+chpl/chpl-resources/src/main/resources/environment.properties
+chpl/chpl-resources/src/main/resources/errors.properties
+chpl/chpl-resources/src/main/resources/lookup.properties
 ```
+Any value in the above files can be overridden by creating an "override" file.  The file should be name `originalFileName-override.properites` and be placed somewhere on the classpath.  In the "override" file, you can redefine any existing key with a new value.
+
+Each of the above files has "default" values for each key, though a few keys within the files have a value of "SECRET".  This applies to certain values that only apply to a particular instance of CHPL - things like server names, usernames, passwords, etc.
+
+| File                   | Key                                | Description                                                                                       |
+|------------------------|------------------------------------|---------------------------------------------------------------------------------------------------|
+| email.properties       | directReview.chplChanges.email     |                                                                                                   |
```

Comment:
Email account to send notification of listing or developer changes that do affect direct reviews